content/en/error_tracking/issue_states.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Issue States and Workflows
+kind: documentation
+disable_toc: false
+---
+
+## Issue status
+All issues in Error Tracking have a status to help you triage and prioritize issues or dismiss noise. There are three statuses:
+- Open
+- Ignored
+- Resolved
+
+Open
+: All issues start in the open status. An open issue is ongoing and needs attention because it's new, it's a regression, or it needs to be fixed. 
+
+Ignored
+: Ignored issues require no additional investigation or action.
+
+Resolved
+: Resolved issues have been fixed. The error no longer occurs.
+
+In the Resolved state, if the same error recurs in a newer version of the code, or the error occurs again in code without versions, Error Tracking triggers a [regression](#regressions). The issue moves to the **Open** state.
+
+<!-- ### UNRESOLVED
```

Comment:
I had to move this comment to the bottom so the description list got formatted correctly.